cmd/apichecker/apicheck.go

```diff
@@ -738,22 +738,13 @@ func htmlScrapeHitBTC(htmlData *HTMLScrapingData) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	aBody := string(a)
+
 	r, err := regexp.Compile(htmlData.RegExp)
```

Comment:
🥃 This isn't coming from outside, so we could `MustCompile` safely.
Could also make htmlData.RegExp into a regexp.Regexp